client.py

```diff
@@ -6,9 +6,15 @@
 logger = logging.getLogger(__name__)
 
 class client:
+    # CLIENT connects with server.py on host machine to control
+    #  varoius pieces of equipment
+    #  Note, some hwserver operations could conveivably take longer than the
+    #  default timeout used here. If so, obj.connection.Timeout should be
+    #  adjusted appropriately
 
     DEFAULT_IP = 'localhost'
     DEFAULT_PORT = 36577
+    DEFAULT_TIMEOUT = 2
 
     def __init__(self,ip=DEFAULT_IP,port=DEFAULT_PORT):
```

Comment:
As per above comment, let's change `ip` to `host` since host can mean host *name* or IP.